### PR TITLE
Fixes #1683.  Correct misleading information around ports 4789 & 7946

### LINKED ISSUE #1683: Error in UCP docs about port 7946
@adrahon commented

In the System Requirements part of the docs for both UCP 2.0 and 2.1 beta, we list ports that need to be open, we have the following entries:
managers, workers	in	TCP 2377 (configurable)	Port for communication between swarm nodes
managers, workers	in, out	TCP, UDP 4789	Port for overlay networking
managers, workers	in, out	TCP, UDP 7946	Port for overlay networking

According to the Docker Engine Swarm docs it's:
TCP port 2377 for cluster management communications
TCP and UDP port 7946 for communication among nodes
TCP and UDP port 4789 for overlay network traffic

My understanding is that:
 - TCP 2377 is for swarm node RPC communication
 - TCP / UDP 7946 is for gossip-based clustering
 - TCP / UDP 4789 for overlay networking

### PR DESCRIPTION
**Pull request by clocklear**
### Proposed changes

Fixed incorrect information around ports 4789 and 7946 in UCP 2.0 / 2.1 documentation

### Related issues (optional)

Fixes #1683

### CODE REVIEW

File: datacenter/ucp/2.0/guides/installation/system-requirements.md [hunk 1]
```diff
@@ -31,8 +31,8 @@ When installing UCP on a host, make sure the following ports are open:
 | managers, workers |    in     | TCP 443  (configurable) | Port for the UCP web UI and API                                                   |
 | managers          |    in     | TCP 2376 (configurable) | Port for the Docker Swarm manager. Used for backwards compatibility               |
 | managers, workers |    in     | TCP 2377 (configurable) | Port for communication between swarm nodes                                        |
-| managers, workers |  in, out  | UDP 4789                | Port for overlay networking                                                       |
-| managers, workers |  in, out  | TCP, UDP 7946           | Port  for overlay networking                                                      |
+| managers, workers |  in, out  | TCP, UDP 4789           | Port for overlay networking                                                       |
```
Comment (joaofnfernandes) [inline datacenter/ucp/2.0/guides/installation/system-requirements.md]: According to the [swarm docs](https://docs.docker.com/engine/swarm/swarm-tutorial/#the-ip-address-of-the-manager-machine) port 4789 is just UDP, so I'm thinking that #1683 is probably wrong on that one. Can you remove the mention to TCP?

Comment (clocklear) [inline datacenter/ucp/2.0/guides/installation/system-requirements.md]: Sure thing.  Give me a few mins.

**Comment by joaofnfernandes** [comment]
All the feedback was addressed, it's time to ship 🚀

**Comment by clocklear** [comment]
Wooo-hoo! Thanks!

**Pull request merged**